crates/node-core/src/args/gas_price_oracle.rs

```diff
@@ -1,6 +1,6 @@
 use crate::primitives::U256;
 use clap::Args;
-use reth_rpc::eth::gas_oracle::GasPriceOracleConfig;
+use reth_rpc_eth_api::GasPriceOracleConfig;
```

Comment:
I want to avoid these changes, instead this should be re-exported from reth-rpc

Reply:
`eth` is the only `reth-rpc` module used in `reth-node-core` builder, hence why I went with updating the dep here